pkg/auth/authenticator.go

```diff
@@ -221,19 +293,27 @@ func (a *Authenticator) Check(ctx context.Context, request *Request) (*Response,
 		response.Header.Add(key, value)
 	}
 
+	// update metrics
+	reqCount.With(ReasonLabel(reason)).Inc()
+	reqLatency.With(ReasonLabel(reason)).Observe(time.Since(reqStartTime).Seconds())
```

Comment:
Shouldn't we calculate request time based on Check func inside server.go
([here](https://github.com/snapp-incubator/Cerberus/blob/2549ff38ba71b46f07d2c8f67cd10ca22e59d86f/pkg/auth/server.go#L33)) 
so we can include the conversion time too?

Reply:
That's a good Idea. I'll change it

Reply:
resolved in https://github.com/snapp-incubator/Cerberus/pull/13/